### PR TITLE
show processlist can check sleep connection

### LINKED ISSUE #20896: show processlist shows sleep connections
## Feature Request

SS currently show processlist can only display the connection in execution, but not the connection in the sleep. I hope to be able to display all the same as MySQL, if I need to filter the sleep, can use the following command.

```
pager grep -v sleep;show processlist;
```

### PR DESCRIPTION
**Pull request by natehuangting**
Fixes #20896

### CODE REVIEW

**Review by tuichenchuxin — CHANGES_REQUESTED** [review]
This pr will show the proxy front sleep connection? 
But curent show processlist logic show the backend execution.

**Comment by natehuangting** [comment]
> This pr will show the proxy front sleep connection? But curent show processlist logic show the backend execution.

It will show all the connection status, including the background sleep in execution.

```
mysql> show processlist;
+--------------------------------------+------+-----------+-------------+---------+------+-------+------+
| Id                                   | User | Host      | db          | Command | Time | State | Info |
+--------------------------------------+------+-----------+-------------+---------+------+-------+------+
| cb9e262f-2833-4925-2d8b-f905237c58c6 | root | 127.0.0.1 | sharding_XA | Sleep   | 14   |       |      |
+--------------------------------------+------+-----------+-------------+---------+------+-------+------+
1 row in set (0.02 sec)

mysql> show processlist;
+--------------------------------------+------+-----------+-------------+---------+------+---------------+-----------------------+
| Id                                   | User | Host      | db          | Command | Time | State         | Info                  |
+--------------------------------------+------+-----------+-------------+---------+------+---------------+-----------------------+
| cb9e262f-2833-4925-2d8b-f905237c58c6 | root | 127.0.0.1 | sharding_XA | Sleep   | 40   |               |                       |
| fc646062-99fc-2556-d42e-257dc7c9c621 | root | 127.0.0.1 | sharding_xa | Execute | 2    | Executing 0/2 | select * from t_order |
+--------------------------------------+------+-----------+-------------+---------+------+---------------+-----------------------+
2 rows in set (0.03 sec)
```

**Comment by tuichenchuxin** [comment]
Looks great!

File: shardingsphere-mode/shardingsphere-mode-core/src/main/java/org/apache/shardingsphere/mode/process/GovernanceExecuteProcessReporter.java [hunk 1]
```diff
@@ -54,7 +54,13 @@ public void report(final String executionID, final ExecuteProcessConstants const
     
     @Override
     public void reportClean(final String executionID) {
-        ShowProcessListManager.getInstance().removeProcessContext(executionID);
```
Comment (tuichenchuxin) [inline shardingsphere-mode/shardingsphere-mode-core/src/main/java/org/apache/shardingsphere/mode/process/GovernanceExecuteProcessReporter.java:57]: remove this line may result in jdbc mode, not clear process context.

**Review by tuichenchuxin — CHANGES_REQUESTED** [review]

**Review by tuichenchuxin — DISMISSED** [review]
LGTM

File: shardingsphere-infra/shardingsphere-infra-executor/src/main/java/org/apache/shardingsphere/infra/executor/kernel/model/ExecutionGroupContext.java [hunk 2]
```diff
@@ -38,7 +38,7 @@
     
     private final Collection<ExecutionGroup<T>> inputGroups;
     
-    private final String executionID = new UUID(ThreadLocalRandom.current().nextLong(), ThreadLocalRandom.current().nextLong()).toString().replace("-", "");
+    private String executionID = new UUID(ThreadLocalRandom.current().nextLong(), ThreadLocalRandom.current().nextLong()).toString().replace("-", "");
```
Comment (strongduanmu) [inline shardingsphere-infra/shardingsphere-infra-executor/src/main/java/org/apache/shardingsphere/infra/executor/kernel/model/ExecutionGroupContext.java:41]: Can we move `new UUID` to proxy and jdbc adaptor?

File: shardingsphere-infra/shardingsphere-infra-executor/src/main/java/org/apache/shardingsphere/infra/executor/sql/process/ExecuteProcessEngine.java [hunk 3]
```diff
@@ -41,13 +45,41 @@
...
+     * Initialize connection.
+     *
+     * @param grantee grantee
+     * @param databaseName database name
+     * @param eventBusContext event bus context
+     * @return execution id
+     */
+    public static String initializeConnection(final Grantee grantee, final String databaseName, final EventBusContext eventBusContext) {
+        ExecutionGroupContext<SQLExecutionUnit> executionGroupContext = new ExecutionGroupContext<>(Collections.emptyList());
+        executionGroupContext.setExecutionID(new UUID(ThreadLocalRandom.current().nextLong(), ThreadLocalRandom.current().nextLong()).toString());
```
Comment (strongduanmu) [inline shardingsphere-infra/shardingsphere-infra-executor/src/main/java/org/apache/shardingsphere/infra/executor/sql/process/ExecuteProcessEngine.java:57]: Please add `.replace("-", "")` after `new UUID(ThreadLocalRandom.current().nextLong(), ThreadLocalRandom.current().nextLong()).toString()` single mysql kill processlist statement parse logic doesn't support `-`.

**Review by strongduanmu — CHANGES_REQUESTED** [review]

File: shardingsphere-infra/shardingsphere-infra-executor/src/main/java/org/apache/shardingsphere/infra/executor/kernel/model/ExecutionGroupContext.java (hunk 2 above)
Comment (natehuangting) [reply shardingsphere-infra/shardingsphere-infra-executor/src/main/java/org/apache/shardingsphere/infra/executor/kernel/model/ExecutionGroupContext.java:41]: I would move that logic to the ExecuteProcessEngine.

File: shardingsphere-infra/shardingsphere-infra-executor/src/main/java/org/apache/shardingsphere/infra/executor/sql/process/ExecuteProcessEngine.java (hunk 3 above)
Comment (natehuangting) [reply shardingsphere-infra/shardingsphere-infra-executor/src/main/java/org/apache/shardingsphere/infra/executor/sql/process/ExecuteProcessEngine.java:57]: Yeah!

**Review by strongduanmu — CHANGES_REQUESTED** [review]
Can you fix ci error?

**Review by strongduanmu — APPROVED** [review]

**Pull request merged**